sos/report/plugins/networking.py

```diff
@@ -183,14 +183,23 @@ def setup(self):
                 "ethtool -a " + eth,
                 "ethtool -c " + eth,
                 "ethtool -g " + eth,
-                "ethtool -e " + eth,
                 "ethtool -P " + eth,
                 "ethtool -l " + eth,
                 "ethtool --phy-statistics " + eth,
                 "ethtool --show-priv-flags " + eth,
                 "ethtool --show-eee " + eth
             ])
 
+            # skip EEPROM collection for 'bnx2x' driver NICs as this command 
+            # can pause the NIC and is not production safe.
+            cmd="ethtool -i {}".format(eth)
+            driver = self.get_command_output(cmd)
+            if not "bnx2x" in driver['output']:
+                self.add_cmd_output(["ethtool -e" + eth])
+            else:
+                self._log_warn("skipped command 'ethtool -e' for 'bnx2x' interface {} as it may pause the NIC for EEPROM collection.".format(eth))
```

Comment:
This line is way too long, it needs to be wrapped under 80 characters per PEP8.

Also a warning will automatically be produced if you use a `SoSPredicate` instead.